sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -314,16 +314,17 @@ public EncryptedBlobClientBuilder connectionString(String connectionString) {
 
         Map<String, String> connectionStringPieces = Utility.parseConnectionString(connectionString);
 
-        String accountName = connectionStringPieces.get(Constants.ConnectionStringConstants.ACCOUNT_NAME);
-        String accountKey = connectionStringPieces.get(Constants.ConnectionStringConstants.ACCOUNT_KEY);
+        String accountName = connectionStringPieces.get(Constants.ConnectionStringConstants.ACCOUNT_NAME_NAME);
+        String accountKey = connectionStringPieces.get(Constants.ConnectionStringConstants.ACCOUNT_KEY_NAME);
 
         if (ImplUtils.isNullOrEmpty(accountName) || ImplUtils.isNullOrEmpty(accountKey)) {
             throw logger.logExceptionAsError(
                 new IllegalArgumentException("'connectionString' must contain 'AccountName' and 'AccountKey'."));
         }
 
-        String endpointProtocol = connectionStringPieces.get(Constants.ConnectionStringConstants.ENDPOINT_PROTOCOL);
-        String endpointSuffix = connectionStringPieces.get(Constants.ConnectionStringConstants.ENDPOINT_SUFFIX);
+        String endpointProtocol
```

Comment:
Why do we use different way of constructing endpoint here compare to blob, file..?

Reply:
yes, I'm about to update the PR to use `StorageConnectionString` type here as well that also removes parseConnectionString from Utility.